zone-service/src/main/java/org/cloudfoundry/identity/uaa/zone/OrchestratorZoneService.java

```diff
@@ -296,10 +296,16 @@ private IdentityZone createIdentityZone(IdentityZone identityZone)
             logger.debug("Zone - creating zone name [" + identityZone.getName() + "]");
             created = zoneProvisioning.create(identityZone);
             logger.debug("Zone - created zone name [" + identityZone.getName() + "]");
+        } catch (ZoneAlreadyExistsException e) {
+            String errorMessage =  "The \"" + UAA_CUSTOM_SUBDOMAIN
+                                   + "\" name is already taken. Please use a different subdomain.";
```

Comment:
Add subdomain name to the error message.  
`String errorMessage = String.format("The subdomain name %s is already taken. Please use a different subdomain",  identityZone.getSubdomain());`

Reply:
Done, Thanks!